### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by jwilk**
Found using [anorack](https://github.com/jwilk/anorack).

### CODE REVIEW

File: messages.c [hunk 1]
```diff
@@ -122,7 +122,7 @@ static const char *warning_strings[] = {
 	"W: Unknown extended key usage\n", /* WARN_UNKNOWN_EKU */
 	"W: RSA public exponent not in range of 2^16+1 to 2^256-1\n", /* WARN_RSA_EXP_RANGE */
 	"W: Policy information has qualifier other than CPS URI\n", /* WARN_POLICY_QUALIFIER_NOT_CPS */
-	"W: explicitText is not using an UTF8String\n", /* WARN_EXPLICIT_TEXT_ENCODING */
+	"W: explicitText is not using a UTF8String\n", /* WARN_EXPLICIT_TEXT_ENCODING */
```
Comment (kroeckx) [inline messages.c]: This sounds correct to me?

Comment (kroeckx) [inline messages.c]: It's seems it's "a" before a "u".

**Comment by jwilk** [comment]
The choice of the indefinite article depends on pronunciation of the preceding word.
I believe that "UTF" is supposed to be pronounced roughly like "you-tea-ef"; so it starts with a consonant sound /j/, which warrants the "a" article.
Just to be sure, I checked which one the Unicode standard uses: there's 30 instances of "a UTF", and no instances of "an UTF".

File: messages.c (hunk 1 above)
Comment (robstradling) [inline messages.c]: Yes.
FYI: http://english.stackexchange.com/questions/252217/why-is-it-a-unicorn

**Pull request merged**